### PR TITLE
[ng] src reorg of architecture example

### PR DESCRIPTION
**Pull request by chalin**
Contributes to #713

### CODE REVIEW

File: src/angular/guide/architecture.jade [hunk 1]
```diff
@@ -99,10 +99,10 @@ block registering-providers
     In general, add providers when bootstrapping the application so that
     the same instance of a service is available everywhere.
 
-  +makeExcerpt('app/main.ts', 'bootstrap')
+  +makeExcerpt('web/main.dart (bootstrap)')
```
Comment (kwalrath) [inline src/angular/guide/architecture.jade]: The new excerpt doesn't show adding providers.

File: examples/ng/doc/architecture/lib/app_component.dart [hunk 2]
```diff
@@ -2,14 +2,20 @@
...
-      <sales-tax></sales-tax>''',
-    directives: const [HeroListComponent, SalesTaxComponent])
-// #docregion export
+  selector: 'my-app',
+  template: '''
+    <hero-list></hero-list>
+    <sales-tax></sales-tax>
+  ''',
+  directives: const [HeroListComponent, SalesTaxComponent],
+  providers: const [BackendService, HeroService, Logger],
```
Comment (kwalrath) [inline examples/ng/doc/architecture/lib/app_component.dart]: Maybe this is what we should feature for showing how to define common providers?

**Comment by chalin** [comment]
Good catch.

I've updated the page from:
> screen shot 2017-06-13 at 3 58 20 pm

to
> screen shot 2017-06-13 at 4 03 39 pm

WDYT?

**Comment by kwalrath** [comment]
Wow, has the recommendation really changed so completely? It seems to me that AppComponent is the new bootstrap...

File: examples/ng/doc/architecture/lib/app_component.dart (hunk 2 above)
Comment (chalin) [inline examples/ng/doc/architecture/lib/app_component.dart]: Done

**Pull request merged**